TestForSignUp/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OpenQA.Selenium.Chrome;
```

Comment:
`Unhandled exception. System.InvalidOperationException: session not created: This version of ChromeDriver only supports Chrome version 81 (SessionNotCreated)` HM

Reply:
Ok, my bad having not updated chrome, but better to add some handler

Reply:
added